domains/_discord.link_trzz.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "vanilla-dd",
+           "email": "119125583+vanilla-dd@users.noreply.github.com",
```

Comment:
please either change your email or leave the email line blank

Reply:
done